docs/src/pages/configurations/custom-webpack-config/index.md

```diff
@@ -253,6 +253,8 @@ const path = require('path');
 const custom = require('../webpack.config.js');
 
 module.exports = async ({ config, mode }) => {
-  return { ...config, loaders: custom.loaders };
+  config.module.rules.push(...custom.module.rules);
```

Comment:
Thanks for the PR! However, I don't think this is correct.

The old example replaces the loaders with your app's loaders.

Your change appends the loaders to the default loaders provided by storybook.

I think the intent of the example is to use your existing webpack config, which means replacing Storybook's loaders entirely.

Let me know what you think!

cc @ndelangen @igor-dv

Reply:
Thanks for Review! I agree with you and update PR soon.

Reply:
I updated a PR, review again please.

Reply:
Aha I didn't see the bug in the original. Good catch and thanks for following up on this! ❤️